### Module 5: Basic Stats and Probability with Python

##### Summary Statistics

Assignment:

1) Greg was 14, Marcia was 12, Peter was 11, Jan was 10, Bobby was 8, and Cindy was 6 when they started playing the Brady kids on The Brady Bunch. Cousin Oliver was 8 years old when he joined the show. What are the mean, median, and mode of the kids' ages when they first appeared on the show? What are the variance, standard deviation, and standard error?

2) Using these estimates, if you had to choose only one estimate of central tendency and one estimate of variance to describe the data, which would you pick and why?

3) Next, Cindy has a birthday. Update your estimates- what changed, and what didn't?

4) Nobody likes Cousin Oliver. Maybe the network should have used an even younger actor. Replace Cousin Oliver with 1-year-old Jessica, then recalculate again. Does this change your choice of central tendency or variance estimation methods?

5) On the 50th anniversary of The Brady Bunch, four different magazines asked their readers whether they were fans of the show. The answers were: TV Guide 20% fans Entertainment Weekly 23% fans Pop Culture Today 17% fans SciPhi Phanatic 5% fans

Based on these numbers, what percentage of adult Americans would you estimate were Brady Bunch fans on the 50th anniversary of the show?

In [10]:
import pandas as pd
import numpy as np
import statistics

##### Question 1

In [1]:
#Creat a list of the ages in order to find the summary statistics of the Brady kids' age...

age = [14, 12, 11, 10, 8, 6, 8]

In [7]:
#Find the mean...
np.mean(age)

9.857142857142858

In [8]:
#Find the median...
np.median(age)

10.0

In [11]:
#Finde the mode...
statistics.mode(age)

8

In [12]:
#find the variance...
np.var(age)

6.408163265306122

In [13]:
#find the standard deviation...
np.std(age)

2.531435020952764

In [14]:
#what does changing the 'ddof' do?
np.std(age, ddof = 1)

2.734262327610589

In [15]:
#find the standard error...
np.std(age)/np.sqrt(len(age))

0.9567925036515135

##### Qestion 2

If I had to choose only one estimate of the Central Tendency to describe the data I would choose the mean. I would choose the mean because I feel as if it gives the best representation of the average age, although it is not significantly different than the mode. However, I do feel as if 9.8 is closer to the average age than 10.0.

If I had to choose one estimate to describe the variance of the data then I would choose the variance measurement itself. I would choose this because I think that the high variance value demonstrates that there is a wide array in the age of the children.

##### Question 3

In [21]:
age2 = [14, 12, 11, 10, 8, 7, 8]

In [22]:
np.mean(age2)

10.0

In [23]:
np.median(age2)

10.0

In [24]:
statistics.mode(age2)

8

In [25]:
np.var(age2)

5.428571428571429

In [26]:
np.std(age2)

2.32992949004287

In [28]:
np.std(age2)/np.sqrt(len(age2))

0.8806305718527109

The most notable difference in the summary statistics is that the mean increases. The mode and the median remain the same.

The most notable change in the variance was the change again in the variance value. I was surprised to see that a change of only one year decreases the entire variance by more than one. However, that does make sense as the age was so substantially below the rest of the years. Therefore we can understand why it would have a greater impact on decreasing the variance in the overall varaince measure of age.

##### Question 4

In [29]:
age3 = [14, 12, 11, 10, 8, 7, 1]

In [30]:
np.mean(age3)

9.0

In [31]:
np.median(age3)

10.0

In [32]:
statistics.mode(age3)

StatisticsError: no unique mode; found 7 equally common values

In [33]:
np.var(age3)

15.428571428571429

In [34]:
np.std(age3)

3.927922024247863

In [35]:
np.std(age3)/np.sqrt(len(age3))

1.4846149779161806

I still find that the most descriptive measurements are the mean and the variance. They demonstrate the breadth of the observations more succinctly than any of the other measurements and show how drastic the change is from having an 8 year old to a 1 year old.

In [36]:
fans = [20, 23, 17, 5]

In [37]:
np.mean(fans)

16.25

Based off of the given measurements, I would guess that %16.25 of Americans were fans of the Brady Brunch.